runtime/syntax/config.vim

```diff
@@ -34,6 +34,9 @@ syn region  configstring    start=+\z(["'`]\)+ skip=+\\\z1+ end=+\z1+ contains=@
 syn region  configmsg matchgroup=configfunction start="AC_MSG_[A-Z]*\ze(\[" matchgroup=configdelimiter end="\])" contains=configdelimiter,@Spell
 syn region  configmsg matchgroup=configfunction start="AC_MSG_[A-Z]*\ze([^[]" matchgroup=configdelimiter end=")" contains=configdelimiter,@Spell
 
+" Help write portable shell code
+syn match acPlusEq '\v\+\=' containedin=ALLBUT,configcomment
```

Comment:
This group name is inconsistent with the others. All other group names starts with `config`.